app/presenters/summary/sections/contact_details.rb

```diff
@@ -24,12 +24,10 @@ def answers
                        ))
         end
 
-        unless residence_of_type?('none')
-          answers.push(Components::FreeTextAnswer.new(
-                         :home_address, full_address(home_address), show: true,
-            change_path: change_path(home_address)
-                       ))
-        end
+        answers.push(Components::FreeTextAnswer.new(
+                       :home_address, full_address(home_address), show: show_home_address?,
+          change_path: change_path(home_address)
+                     ))
```

Comment:
This was always displaying as show was set to true but the logic is to always display for historical applications but not display when `residence type == none`